stdlib/os/__init__.pyi

```diff
@@ -1162,3 +1162,17 @@ if sys.version_info >= (3, 13) and sys.platform != "win32":
     def grantpt(fd: FileDescriptorLike, /) -> None: ...
     def unlockpt(fd: FileDescriptorLike, /) -> None: ...
     def ptsname(fd: FileDescriptorLike, /) -> str: ...
+
+if sys.version_info >= (3, 13) and sys.platform == "linux":
+    TFD_TIMER_ABSTIME: Final = 1
```

Comment:
Should be consistent across environments. See: https://github.com/torvalds/linux/blob/2c9b3512402ed192d1f43f4531fb5da947e72bd0/include/uapi/linux/timerfd.h#L31